### DISCUSSION TITLE
Potential updates

### DISCUSSION THREAD

**Discussion by DigimonTrainer** [to2mbn/skinview3d#138] [Ideas]
Is it possible, as an update to the viewer, to have a swimming animation added to it? And possibly more Panorama backdrops? Just a suggestion.

**Comment by undefined303** [comment]
Tried to write it..It's really really hard to me.Mainly because I didn't have enough time, I gave up

**Comment by undefined303** [comment]
> Is it possible, as an update to the viewer, to have a swimming animation added to it? And possibly more Panorama backdrops? Just a suggestion.

swimming animation #203
